src/fides/api/ctl/routes/health.py

```diff
@@ -72,7 +72,11 @@ async def health(
     db: Session = Depends(get_db),
 ) -> Dict:  # Intentionally injecting the ops get_db
     """Confirm that the API is running and healthy."""
-    database_health = get_db_health(CONFIG.database.sync_database_uri, db)
+    database_health = (
+        "No database uri provided"
+        if not CONFIG.database.sync_database_uri
+        else get_db_health(CONFIG.database.sync_database_uri, db=db)
```

Comment:
`if not` conditions are a pet peeve:
```suggestion
        get_db_health(CONFIG.database.sync_database_uri, db=db)
        if CONFIG.database.sync_database_uri is not None
        else "No database uri provided"
```